src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,100 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.exception.UserAgeException;
+import core.basesyntax.exception.UserLoginExistsException;
+import core.basesyntax.exception.UserLoginLengthException;
+import core.basesyntax.exception.UserPasswordLengthException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private final RegistrationService registrationService = new RegistrationServiceImpl();
```

Comment:
Consider using `@BeforeEach` or `@BeforeAll` to initialize your `RegistrationServiceImpl` class instance. `@BeforeEach` creates a new instance before each test, ensuring test isolation. `@BeforeAll` creates a single instance for all tests, which is not suitable if you need to reset the state before each test.